chore(docs): update hosting

Michael-N-M opened the pull request:
This update focuses the topic on hosting NLU services required to run Botpress.

Michael-N-M committed:
update hosting.md

This update focuses the topic on hosting NLU services required to run Botpress.

Michael-N-M committed:
update database.md

EFF commented on docs/guide/docs/infrastructure/database.md:
```diff
@@ -1,27 +1,27 @@
 ---
 
-Botpress comes with support for SQL databases out-the-box and can be accessed by:
+Botpress comes with support for SQL databases out-the-box. These can be accessed using the following methods:
```
can we just remove that section ? from line 6 to 19 I'd like if we simply remove that as I feel it's more confusing than valuable

Michael-N-M replied:
Okay I agree, let me fix that.

Michael-N-M committed:
EFF recomendation

I thought it would be better to restructure the way the information is presented rather than removing it altogether. This doc pointed me to the resources needed to access databases from within actions so maybe someone in the future might benefit.

Michael-N-M committed:
database.md further slimming
